### DISCUSSION TITLE
Control keyboard on mobile devices

### DISCUSSION THREAD

**Discussion by ositoMalvado** [flet-dev/flet#4107] [Ideas]
I would like to be able to open and close the keyboard of my mobile device (be it android or ios) from a simple function call. I imagine something like:

page.close(keyboard)
page.open(keyboard)

Because when I use TextField with Multiline there is no way to hide the device keyboard without clicking another control or disabling TextField (what makes weird colors or handling the TextField) Sorry for my bad english, is this a good idea?

**Comment by Michel7GitHub** [comment]
I've got the same issue

**Reply by Muddassir-Farooq-official** [reply]
Same issue here

**Reply by fzbian** [reply]
any solution?

**Reply by Laztef** [reply]
You can simulate a blur adding an function on the Container father of your Textfield.
¡Try this!
